bulk_site_creator/views.py

```diff
@@ -238,15 +271,40 @@ def create_bulk_job(request: HttpRequest) -> Optional[JsonResponse]:
+                                                                                                                                                          "user_id": user_id,
+                                                                                                                                                          "user_full_name": user_full_name,
+                                                                                                                                                          "user_email": user_email,
+                                                                                                                                                          "school": school_id,
+                                                                                                                                                          "term_id": term_id,
+                                                                                                                                                          "term_name": term_name,
+                                                                                                                                                          "department_id": department_id,
+                                                                                                                                                          "department_name": department_name,
+                                                                                                                                                          "course_group_id": course_group_id,
+                                                                                                                                                          "course_group_name": course_group_name,
+                                                                                                                                                          "template_id": template_id
+                                                                                                                                                          })
+
             # Write the TaskRecords to DynamoDB. We insert these first since the subsequent JobRecord
             # kicks off the downstream bulk workflow via a DynamoDB stream.
             batch_write_item(dynamodb_table, tasks)
 
             # Now write the JobRecord to DynamoDB
             response = dynamodb_table.put_item(Item=job.to_dict())
             if response["ResponseMetadata"]["HTTPStatusCode"] != 200:
-                logger.error(f"Error adding JobRecord to DynamoDB: {response}")
-                # TODO improve this logging statement
+                logger.error(f"Error adding JobRecord to DynamoDB: {response}", extra={"sis_account_id": sis_account_id,
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

You have a log statement that might use unsanitized input. Depending on the context, this could result in:

1. A log injection attack that breaks log integrity, forges log entries, or bypasses monitors that use the logs. To increase the security of your code, sanitize your inputs before logging them. [Learn more](https://cwe.mitre.org/data/definitions/117.html)

2. A sensitive information leak that exposes users&#39; credentials, private information, or identifying information to an attacker. To preserve privacy in your code, redact sensitive user information before logging it. [Learn more](https://cwe.mitre.org/data/definitions/532.html)

Similar issue at line number 371.